soco/core.py

```diff
@@ -960,6 +961,48 @@ def dialog_mode(self, dialog_mode):
             ]
         )
 
+    @property
+    def trueplay(self):
+        """bool: Whether Trueplay is enabled on this device.
+        True if on, False if off.
+
+        Devices that do not support Trueplay, or which do not have
+        a current Trueplay calibration, will raise a `NotSupportedException`
+        when both getting and setting the property.
+
+        Can only be set on visible devices. Attempting to set on non-visible
+        devices will raise a `SoCoNotVisibleException`.
+        """
+        response = self.renderingControl.GetRoomCalibrationStatus([("InstanceID", 0)])
+        if response["RoomCalibrationAvailable"] == "0":
+            raise NotSupportedException
+        return response["RoomCalibrationEnabled"] == "1"
```

Comment:
```suggestion
        if response["RoomCalibrationAvailable"] == "0":
            raise NotSupportedException
            
        return response["RoomCalibrationEnabled"] == "1"
```
Separating the error handling makes the code cleaner.

A question, should this rather return `None` when the trueplay calibration is not done? This way it would be consistent with the handling of some other features (e.g., night_mode, dialog_mode which only raise an exception in setters).

Reply:
Thanks. Yes, I think this is better, and I've made the change.